main.nf

```diff
@@ -62,6 +62,8 @@ def helpMessage() {
 
     Assembly:
       --coassemble_group [bool]             Co-assemble samples within one group, instead of assembling each sample separately (default: false)
+      --spades_options [string]             Additional custom options for SPAdes, e.g. "-k 21,33,55,77" or from https://github.com/ablab/spades#advanced-options
```

Comment:
important: other than `-t`, `-m` `-o` and `-out-prefix` !

Reply:
ups, this particular options were meant for megahit

Reply:
True, I'll add that info for megahit and spades

Reply:
done